src/hotspot/share/opto/loopnode.hpp

```diff
@@ -992,6 +992,10 @@ class PhaseIdealLoop : public PhaseTransform {
     assert( ctrl == find_non_split_ctrl(ctrl), "must set legal crtl" );
     _loop_or_ctrl.map(n->_idx, (Node*)((intptr_t)ctrl + 1));
   }
+  void set_root_as_ctrl(Node* n) {
+    assert( !has_node(n) || has_ctrl(n), "" );
```

Comment:
We don't use spaces after and before `()` in assert(). Ignore old style in previous lines.